extensions/amp-stream-gallery/1.0/README.md

```diff
@@ -0,0 +1,438 @@
+
+```html
+<head>
+  <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
+  <script async src="https://cdn.ampproject.org/v0/bento-stream-gallery-1.0.js"></script>
+</head>
+<body>
+  <bento-stream-gallery>
+    <img src="img1.png">
+    <img src="img2.png">
+    <img src="img3.png">
+    <img src="img4.png">
+    <img src="img5.png">
+    <img src="img6.png">
+    <img src="img7.png">
+  </bento-stream-gallery>
+  <script>
+    (async () => {
+      const streamGallery = document.querySelector('#my-stream-gallery');
+      await customElements.whenDefined('bento-stream-gallery');
+      const api = await streamGallery.getApi();
+
+      // programatically expand all sections
+      api.next();
+      // programatically collapse all sections
+      api.prev();
+      // programatically go to slide
+      api.goToSlide(4);
+    })();
+  </script>
+</body>
+```
+
+[/example]
+
+#### Interactivity and API usage
+
+Bento enabled components in standalone use are highly interactive through their API. The `bento-stream-gallery` component API is accessible by including the following script tag in your document:
+
+```javascript
+await customElements.whenDefined('bento-stream-gallery');
+const api = await streamGallery.getApi();
+```
+
+##### Actions
+
+**next()**
+
+Moves the carousel forwards by number of slides visible.
+
+```js
+api.next();
+```
+
+**prev()**
+
+Moves the carousel backwards by number of slides visible.
+
+```js
+api.prev();
+```
+
+**goToSlide(index: number)**
+
+Moves the carousel to the slide specified by the `index` argument.
+Note: `index` will be normalized to a number greater than or equal to `0` and less than the number of slides given.
+
+```js
+api.goToSlide(0); // Advance to first slide.
+api.goToSlide(length - 1); // Advance to last slide.
+```
+
+##### Events
+
+The Bento Stream Gallery component allows you to register and respond to the following events:
+
+**slideChange**
+
+This event is triggered when the index displayed by the carousel has changed.
+The new index is available via `event.data.index`.
+
+```js
+streamGallery.addEventListener('slideChange', (e) => console.log(e.data.index))
+```
+
+#### Layout and style
+
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-streamGallery-1.0.css">
+```
+
+Alternatively, you may also make the light-weight pre-upgrade styles available inline:
+
+```html
+<style data-bento-boilerplate>
+  amp-stream-gallery {
+    display: block;
+    overflow: hidden;
+    position: relative;
+  }
+  amp-stream-gallery:not(.i-amphtml-built) > :not([placeholder]):not(.i-amphtml-svc) {
+    display: none;
+    content-visibility: hidden;
+  }
```

Comment:
Hmm I don't think this one is needed because `i-amphtml-built` is specific to AMP

Reply:
removed